packages/api/src/controllers/wallets.controller.ts

```diff
@@ -28,4 +30,10 @@ export class WalletsController {
 
         return this.walletsService.findAll(getWalletsReq);
     }
+
+    @Delete("/users/:userId/deleteWallet")
```

Comment:
API path standards are important, also make sure you add the guard in

```suggestion
    @UseGuards(JwtAuthGuard, CanMutateResourceGuard)
    @Delete("/users/:userId/wallets/:walletId")
```